README.md

```diff
@@ -27,3 +27,12 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+Answer:
+A good invariant would be at the start for each recursive call to fib(n), the fibonacci numbers for indices 0 to n - 1.
```

Comment:
What about the Fibonacci numbers? An invariant is a statement that can be true or false.